Kitodo/src/main/java/org/kitodo/services/data/TaskService.java

```diff
@@ -699,16 +688,6 @@ private void abortTask(Task task) throws DataException {
         save(task);
     }
 
-    /**
-     * Returns whether this is a step of a process that is part of at least one
-     * batch as read-only property "batchSize". //TODO: is it ever used?
-     *
-     * @return whether this step’s process is in a batch
-     */
-    public boolean isBatchSize(Task task) {
```

Comment:
There are references in `Kitodo/src/main/webapp/pages/tasks.xhtml`.

Reply:
Item with batchSize is for this case TaskDTO. So more likely it is needed to add this property to TaskDTO class... I will look into it closer.